yarn.lock

```diff
@@ -4906,6 +4906,20 @@ babel-extract-comments@^1.0.0:
   dependencies:
     babylon "^6.18.0"
 
+babel-jest@26.6.3, babel-jest@^26.6.0, babel-jest@^26.6.3:
```

Comment:
This is here because I forgot to update the yarn lock when I reverted the Jest version in legacy.